src/main/scala/rps/GameController.scala

```diff
@@ -17,11 +16,17 @@ trait GameController {
 }
 
 class GameControllerImpl(gameService: GameService)(implicit ec: ExecutionContext) extends GameController {
-  override def result(): Future[Either[Throwable, PlayResponse]] = {
-    gameService.getResult.map(p => p.flatMap(_.toRight(new IllegalStateException).map {
-      play => PlayResponse(play.userMove, play.computerMove, play.result)
-    }))
-  }
+  val runtime = Runtime.default
+
+  private[this] def resultZIO = gameService
+    .getResult()
+    .map(_.getOrElse(throw new IllegalStateException))
+    .map(play => PlayResponse(play.userMove, play.computerMove, play.result))
+    .either
+
+  private[this] def playZIO(userMove: Move) = gameService.playMove(userMove).either
```

Comment:
Can you please annotate these methods? It makes it easier to understand (and this is a learning repo, so it's especially important)